tests/sentry/utils/test_validators.py

```diff
@@ -58,3 +58,19 @@ def test_is_span_id():
     assert not is_span_id(4711)
     assert not is_span_id(False)
     assert not is_span_id(None)
+
+
+def test_is_email():
+    assert is_email("abc123@gmail.com")
+    assert is_email("abcde@fgh.ijk.lm")
+    assert is_email("first.last@sentry.io")
+    assert is_email("abc-def_ghi@1234.com")
+    assert is_email("*@sentry.io")
```

Comment:
Hmm, not sure I would call this `is_email` since it allows wildcards. If this were a general validator, I would expect this to be false.

Reply:
updated the name to is_email_search